plugins/vli/fu-vli-pd-device.c

```diff
@@ -629,8 +669,16 @@ fu_vli_pd_device_detach(FuDevice *device, FuProgress *progress, GError **error)
 	if (!fu_vli_pd_device_write_gpios(self, error))
 		return FALSE;
 
+	/* disable charging */
+	if (!fu_vli_pd_device_read_reg(self, 0x0003, &tmp, error))
```

Comment:
same, does 0x0003 have a _name_?

Reply:
How about

`#define FU_VLI_PD_REGISTER_ADDRESS_GPIO_CTL_A   0x0003` 

where CTL is control?

Reply:
Maybe CONTROL_ instead? then that's great.

Reply:
Noted! I'll make the changes and push -Emily